app/assets/stylesheets/aspects.scss

```diff
@@ -4,20 +4,29 @@
     @include selectable-list;
 
     .status_indicator {
-      width: 19px;
+      width: 10px;
       height: 14px;
+
       display: inline-block;
     }
 
     a {
       .text {
-        color: #333333;
+        color: #33333;
+        overflow: scroll;
```

Comment:
Would `overflow: auto` be better here? See the comparison of [`scroll`](https://www.w3schools.com/cssref/playit.asp?filename=playcss_overflow&preval=scroll) versus [`auto`](https://www.w3schools.com/cssref/playit.asp?filename=playcss_overflow&preval=auto).